win32/src/winapi/advapi32.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(non_snake_case)]
+#![allow(non_snake_case, unused_variables)]
```

Comment:
Why are these needed?

I think the macro can add the suppressions on each dllexport.  That would mean non-exported (non-traced) functions still get unused variable warnings.

Reply:
I mention this in the description, but it’s because the tracing made the arguments always-used before. So now hundreds of warnings get thrown. I think we should fix them instead of ignoring them, but for the sake of the PR size, opting to ignore them at a file level.